CareKit/CareKit.xcodeproj/project.pbxproj

```diff
@@ -1107,6 +1107,7 @@
 		5196C801226F8F8F00F1C2A2 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALLOW_TARGET_PLATFORM_SPECIALIZATION = YES;
```

Comment:
I believe this was added in newer versions of Xcode projects (after the original CareKit project was created) that allows projects to build on arm/intel, etc. Since the original project was built with an old version of Xcode, many of the flags updated when I modified them in Xcode 14.3